src/common/settings.ts

```diff
@@ -157,3 +158,24 @@ export function checkIfConfigurationChanged(e: ConfigurationChangeEvent, namespa
     const changed = settings.map((s) => e.affectsConfiguration(s));
     return changed.includes(true);
 }
+
+export function logDefaultFormatter(): void {
+    getWorkspaceFolders().forEach((workspace) => {
+        let config = getConfiguration('editor', { uri: workspace.uri, languageId: 'python' });
+        if (!config) {
+            config = getConfiguration('editor', workspace.uri);
+            if (!config) {
+                traceInfo('Unable to get editor configuration');
+            }
+        }
+        const formatter = config.get<string>('defaultFormatter', '');
+        traceInfo(`Default formatter is set to ${formatter} for workspace ${workspace.uri.fsPath}`);
+        if (formatter !== EXTENSION_ID) {
+            traceWarn(`Black Formatter is NOT set as the default formatter for workspace ${workspace.uri.fsPath}`);
+            traceWarn('To set Black Formatter as the default formatter, add the following to your settings.json file:');
+            traceWarn(
+                `\n"[python]": {\n    "editor.defaultFormatter": "${EXTENSION_ID}",\n    "editor.formatOnSave": true\n}`,
```

Comment:
@luabud I often see people not realizing that they have set a different formatter as their default formatter. I am adding this to warn users in logs that the formatter is not set to black. Let me know if the text here needs to be better.

Reply:
The text looks great to me! My one suggestion would be to remove formatOnSave as it's not really required to set Black as the default formatter.